admin/flocker-zfs.spec

```diff
@@ -0,0 +1,46 @@
+# Build this with `rpmbuild -bb admin/flocker-zfs.spec` on the target distribution.
+
+Name:		flocker-zfs
+Version:	0
+Release:	1%{?dist}
+Summary:	Metapackage installing known working version of ZFS.
+
+Group:		System Environment/Kernel
+License:	ASL 2.0
+URL:		http://www.clusterhq.com/
```

Comment:
:question: Should there be a URL pointing to the repo where the source for this version can be found?